lib/zinnia.js

```diff
@@ -252,33 +253,33 @@ export async function run ({
   const { signal } = controller
   const childProcesses = []
 
-  for (const { module } of ZINNIA_MODULES) {
-    if (!matchesModuleFilter(module)) continue
+  for (const { subnet } of SUBNETS) {
+    if (!matchesSubnetFilter(subnet)) continue
 
-    // all paths are relative to `moduleBinaries`
+    // all paths are relative to `runtimeBinaries`
     const childProcess = execa(
       zinniadExe,
-      [join(module, 'main.js')],
+      [join(subnet, 'main.js')],
       {
-        cwd: moduleSourcesDir,
+        cwd: subnetSourcesDir,
         env: {
-          STATION_ID,
+          STATION_ID: CHECKER_ID,
```

Comment:
@bajtos we might want to follow up in Zinnia here, renaming the flag, but I see this as low priority

Reply:
Can you open a follow-up issue in the Zinnia repo, please?

Reply:
👍 https://github.com/CheckerNetwork/zinnia/issues/657